examples/api/lib/cupertino/segmented_control/cupertino_sliding_segmented_control.1.dart

```diff
@@ -0,0 +1,85 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
I think creating a separate example which is almost the same as the other one but just shows "isMomentary" is unnecessary. We can add a `CupertinoSwitch` in `examples/api/lib/cupertino/segmented_control/cupertino_sliding_segmented_control.0.dart` to show what the segmented control looks like when `isMomentary` is true.